channels/chan_pjsip.c

```diff
@@ -1043,6 +1056,9 @@ static int chan_pjsip_write_stream(struct ast_channel *ast, int stream_num, stru
 
 static int chan_pjsip_write(struct ast_channel *ast, struct ast_frame *frame)
 {
+	if (frame->frametype == AST_FRAME_TEXT && frame->stream_num != -1) {
+		return chan_pjsip_write_stream(ast, frame->stream_num, frame);
+	}
```

Comment:
I'm also kinda curious if this is needed either.

Reply:
This actually is needed, without it no text is received on the respective other side. The second condition of course can be removed.